guide-ru.md

```diff
@@ -1,10 +1,8 @@
-# Как познать Haskell
+# Как изучить Haskell
 
-Этот документ — рекомендованный путь к изучению языка Haskell, основанный на опыте помощи другим.
+Перед вами оптимальный roadmap для изучения Haskell, построенный на основе опыта помощи другим, список рекомендаций от одного из авторов книги [Haskell Book.](https://haskellbook.com)
 
-## Это руководство доступно на других языках:
-
-- [In English](README.md)
```

Comment:
This is for people who don't speak Russian and might be referred to this document via link. It should be better kept.